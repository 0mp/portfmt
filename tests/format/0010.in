OPTIONS_GROUP_FOO=	B A
