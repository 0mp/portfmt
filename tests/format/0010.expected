OPTIONS_GROUP_FOO=	A B
